utils/xsc/configprofile.go

```diff
@@ -6,20 +6,41 @@ import (
 	clientutils "github.com/jfrog/jfrog-client-go/utils"
 	"github.com/jfrog/jfrog-client-go/utils/log"
 	"github.com/jfrog/jfrog-client-go/xsc/services"
+	"github.com/jfrog/jfrog-client-go/xsc/services/utils"
 )
 
-func GetConfigProfile(xrayVersion, xscVersion string, serverDetails *config.ServerDetails, profileName string) (*services.ConfigProfile, error) {
-	if err := clientutils.ValidateMinimumVersion(clientutils.Xsc, xscVersion, services.ConfigProfileMinXscVersion); err != nil {
-		log.Info("Minimal Xsc version required to utilize config profile is '%s'. All configurations will be induced from provided Env vars and files")
-		return nil, err
+func GetConfigProfileByName(xrayVersion, xscVersion string, serverDetails *config.ServerDetails, profileName string) (*services.ConfigProfile, error) {
+	if !utils.IsXscXrayInnerService(xrayVersion) {
+		// We need to validate xsc version only if we use a Xray version prior to Xsc migration
+		if err := clientutils.ValidateMinimumVersion(clientutils.Xsc, xscVersion, services.ConfigProfileMinXscVersion); err != nil {
+			log.Info(fmt.Sprintf("Minimal Xsc version required to utilize config profile by url is '%s'. All configurations will be induced from provided Env vars and files", services.ConfigProfileByUrlMinXrayVersion))
+			return nil, err
+		}
```

Comment:
```suggestion
	if err := clientutils.ValidateMinimumVersion(clientutils.Xsc, xscVersion, services.ConfigProfileMinXscVersion); err != nil {
		log.Info(fmt.Sprintf("Minimal Xsc version required to utilize config profile name is '%s'. All configurations will be induced from provided Env vars and files", services.ConfigProfileMinXscVersion))
		return nil, err
	}
```
No need to check if `IsXscXrayInnerService` here, inside `CreateXscService` you have a backward compatiable object to handle those things. also this is getting by name, I think there is a copy/paste mistake here, also before the fix...